tests/src/core/testqgslayertree.cpp

```diff
@@ -291,6 +293,46 @@ void TestQgsLayerTree::testCheckStateMutuallyExclusiveEdgeCases()
   delete root3;
 }
 
+void TestQgsLayerTree::testRestrictedSymbolSize()
+{
+  QgsSettings settings;
```

Comment:
Can you add the QCoreApplication::setOrganizationName/QCoreApplication::setOrganizationDomain/QCoreApplication::setApplicationName calls to initTestCase in this class so that running the test doesn't mess up your local settings?